changelog.md

```diff
@@ -51,18 +99,220 @@ deliver information you explicitly requested by default, even though we will
 allow some compatibility configuration option. This improves performance by not
 returning information that you might not need.
 
+#### Smarter resource offer handling
+
+When Marathon needed to launch tasks for multiple apps, it would first launch all needed tasks for the first one
+and then proceeding with the next app. One big deployment could block all others. In this new version, Marathon
+will spread the offers among all apps that currently need to launch tasks.
+
+When Marathon detects new tasks that it has to launch, then it will now explicitly request new offers
+with the `reviveOffers` Mesos scheduler API call. This should result in more speedy task launches. If
+for any reason, you dislike this behavior, you can disable it with `--disable_revive_offers_for_new_apps`
+(not recommended).
+
+When Marathon has no current use for an offer, it will decline the offer for a configurable period. This period is
+configurable. A short duration might lead to resource starvation for other frameworks if you run many frameworks
+in your cluster. You should only need to reduce it if you use `--disable_revive_offers_for_new_apps`.
+
+* `--decline_offer_duration` (Default: 120 seconds) The duration (milliseconds) for which to decline offers by default
+
+#### New task launching tuning parameters
+
+To prevent overloading Mesos itself, you can now restrict how many tasks Marathon launches per time interval.
+By default, we allow 1000 unconfirmed task launches every 30 seconds. In addition, Marathon launches
```

Comment:
The sentence is correct, but the small word `unconfirmed` should be explained more detailed.